### ISSUE TITLE
TypeError: cannot use a string pattern on a bytes-like object

### ISSUE THREAD

**Issue by rocchidavide** [zebpalmer/pyQRZ#3]
Running code of the qrz_example.py I obtain:
`/site-packages/requests/packages/urllib3/connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
Traceback (most recent call last):
  File "test.py", line 16, in <module>
    result = qrz.callsign("w7atc")
  File "/Users/Davide/.pyenv/versions/env_qrz/lib/python3.6/site-packages/qrz/qrz_query.py", line 59, in callsign
    self._get_session()
  File "/Users/Davide/.pyenv/versions/env_qrz/lib/python3.6/site-packages/qrz/qrz_query.py", line 51, in _get_session
    self._session_key = regex.search(r.content).group(1)
TypeError: cannot use a string pattern on a bytes-like object`

**Comment by zebpalmer** [comment]
what version of python are you using?

**Comment by rocchidavide** [comment]
3.6.0

Il 25 mag 2017 6:08 PM, "Zeb Palmer" <notifications@github.com> ha scritto:

what version of python are you using?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/zebpalmer/pyQRZ/issues/3#issuecomment-304050458>, or mute
the thread

**Comment by zebpalmer** [comment]
I'll preface my response with the fact that this is old, ugly code. I wrote it in an afternoon, and it worked for my limited use case, but I haven't touched it in four years... there were a couple contributions two years ago, but it's largely sat untouched. Though, according to the docs, the QRZ api hasn't changed since Sept 2014 either....  This was written for 2.7.x, I had intended it to work in 3.2 (current 3.x at the time) but looking at the code, it won't work in python 3.  

It's pretty rough, but it's a short and simple module, so it shouldn't take much clean it up to get it working in 3.x. That being said, my QRZ subscription has expired, I'll have to renew it and send some time on package over the weekend to get it working for 3.x.
